tf2_tools/scripts/view_frames.py

```diff
@@ -72,7 +100,7 @@ def generate_dot(data):
             root = map['parent']
     dot += 'edge [style=invis];\n'
     dot += ' subgraph cluster_legend { style=bold; color=black; label ="view_frames Result";\n'
-    dot += '"Recorded at time: '+str(rospy.Time.now().to_sec())+'"[ shape=plaintext ] ;\n'
+    dot += '"Recorded at time: '+str(node.get_clock().now())+'"[ shape=plaintext ] ;\n'
```

Comment:
Maybe format this as a float number in seconds instead? Currently it seems to be a tuple?